ui.frontend/src/components/PathPicker.tsx

```diff
@@ -0,0 +1,204 @@
+import { Breadcrumbs, Button, ButtonGroup, Content, Dialog, DialogContainer, Flex, Heading, Item, ListView, Selection, Text } from '@adobe/react-spectrum';
+import Document from '@spectrum-icons/workflow/Document';
+import FileCode from '@spectrum-icons/workflow/FileCode';
+import Folder from '@spectrum-icons/workflow/Folder';
+import Home from '@spectrum-icons/workflow/Home';
+import Project from '@spectrum-icons/workflow/Project';
+import { ReactNode, useCallback, useEffect, useState } from 'react';
+import { apiRequest } from '../utils/api';
+import { AssistCodeOutput } from '../utils/api.types';
+
+enum NodeType {
+  FOLDER = 'nt:folder',
+  ORDERED_FOLDER = 'sling:OrderedFolder',
+  SLING_FOLDER = 'sling:Folder',
+  CQ_PROJECTS = 'cq/projects',
```

Comment:
why cq/projects is needed here... looks unexpected ;)

Reply:
I have it in my repo, so I guess it's real use case :)